### ISSUE TITLE
Make modal dialog accessible

### ISSUE THREAD

**Issue by imorente** [netlify/netlify-identity-widget#37]
Some examples:
http://gdkraus.github.io/accessible-modal-dialog/
https://github.com/edenspiekermann/a11y-dialog

**Comment by bcomnes** [comment]
Also ensure password managers are picking up the fields correctly. I think once we switched to the iframe, some are having a harder time detecting it.

**Comment by jlengstorf** [comment]
I just checked this for keyboard nav and it _is_ keyboard accessible. I'm going to close, but please let us know if we're missing other a11y concerns!
